### DISCUSSION TITLE
Welcome to DiscordBot Discussions!

### DISCUSSION THREAD

**Discussion by JaxkDev** [JaxkDev/DiscordBot#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Vicemi** [comment]
My Errors:

Warning: require_once(phar:///home/container/plugins/DiscordBot_v2_0_0.phar/vendor/autoload.php): failed to open stream: phar error: "vendor/autoload.php" is not a file in phar "/home/container/plugins/DiscordBot_v2_0_0.phar" in phar:///home/container/plugins/DiscordBot_v2_0_0.phar/src/JaxkDev/DiscordBot/Communication/BotThread.php on line 54

Fatal error: require_once(): Failed opening required 'phar:///home/container/plugins/DiscordBot_v2_0_0.phar/vendor/autoload.php' (include_path='.:') in phar:///home/container/plugins/DiscordBot_v2_0_0.phar/src/JaxkDev/DiscordBot/Communication/BotThread.php on line 54

**Reply by JaxkDev** [reply]
You didn't build it correctly, see the README for info.

**Comment by Xiaoming0313883** [comment]
how to get a member's username using sendmessage event??

**Reply by JaxkDev** [reply]
https://github.com/DiscordBot-PMMP/ChatBridge/blob/72711c6707d317581eef9f1af2fb05ae9cb03b1f/src/JaxkDev/ChatBridge/EventListener.php#L151

**Reply by Xiaoming0313883** [reply]
oh thanks, my mistake😅
